test/inductor/test_ck_backend.py

```diff
@@ -96,6 +96,53 @@ def mm(a, b):
             Y = mm(a, b)
             torch.testing.assert_close(Y_compiled, Y)
 
+    @unittest.skipIf(not torch.version.hip, "ROCM only")
+    @unittest.skipIf(config.is_fbcode(), "fbcode requires different CK path setup")
+    @unittest.mock.patch.dict(os.environ, {"PATH": _get_path_without_sccache()})
+    @parametrize("max_autotune_gemm_backends", ("CK",))
+    @parametrize("autotune_in_subproc", (True,))
+    def test_max_autotune_precompile_matmul_dynamic(
+        self, max_autotune_gemm_backends, autotune_in_subproc
+    ):
+        """
+        Test matmul with dynamic shapes
+        """
+
+        torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction = False
```

Comment:
Just wondering if this was disabled to avoid certain ROCm kernels.

Reply:
Well, it's here to avoid numeric mismatches. Since the CK kernels do the computation with fp32 dtype, I just hope this setting enables fp32 accumulation in the aten counterpart

Reply:
I think you're right if tf32 is enabled then should do fp32 accum

Reply:
> Well, it's here to avoid numeric mismatches. Since the CK kernels do the computation with fp32 dtype, I just hope this setting enables fp32 accumulation in the aten counterpart

Btw, this setting doesn't do what you think it does. Aten always does accumulation in fp32. This setting simply allows aten to truncate to fp16 intermittently for things like fp16.